src/main/java/org/jenkinsci/plugins/docker/commons/tools/DockerToolInstaller.java

```diff
@@ -162,7 +163,7 @@ public Object invoke(File f, VirtualChannel channel) throws IOException, Interru
         return install;
     }
 
-    @Extension
+    @Extension @Symbol("dockerIoInstaller")
```

Comment:
> it's really hard to figure out what they should actually be called...

https://github.com/jenkinsci/system-config-dsl-plugin is the only plausible client that I know of for a `@Symbol` on a `ToolInstallerDescriptor`.